src/keyboard-shortcut.service.ts

```diff
@@ -84,6 +84,11 @@ export class KeyboardShortcutsService {
     const keyIdentifier = 'keyIdentifier';
     let key = event.key || event[keyIdentifier] || 'Unidentified';
 
+    // testing this
```

Comment:
we don't want to add anything or change anything in the file we are testing, so lets move this kind of stuff out of here and into the spec file